lib/caps/RKL/dxva2

```diff
@@ -0,0 +1,87 @@
+###
+### Copyright (C) 2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+# https://github.com/intel/media-driver/blob/master/docs/media_features.md
```

Comment:
This URL is for iHD driver features.  Thus, it does not apply to dxva2.  Please remove.

Reply:
Also need to remove this URL from other d3d11/dxva2 caps files, please.